src/browser/documentation/sidebar-guides/intro.tsx

```diff
@@ -79,60 +82,71 @@ const slides = [
     <p></p>
     <p>
       You can view the list of keybinding anytime by running{' '}
-      <code>:help keys</code> or by pressing <b>F1 </b>in the editor to see all
-      editor-specific keybindings.
+      <code>:help keys</code> or by pressing <b>F1 </b>in the Editor to see all
+      Editor-specific keybindings.
     </p>
     <br />
     <video autoPlay loop muted playsInline>
-      <source src="./assets/images/Keystores.mp4" type="video/mp4" />
+      <source src="./assets/images/Keystrokes.mp4" type="video/mp4" />
     </video>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s3">
     <h3>Result frame</h3>
-    <p className="lead">Most recently executed command or Cypher query.</p>
+    <p className="lead">
+      <em>Most recently executed command or Cypher query</em>
+    </p>
     <p>
-      A result frame is created for each command execution, added to the top of
-      the stream to create a scrollable collection in reverse chronological
-      order.
+      A result frame is created for each execution and added to the top of the
+      stream to create a scrollable collection in reverse chronological order.
     </p>
     <ul>
-      <li>If a frame is pinned it will always stay in the same position. </li>
+      <li>A pinned frame always stays in the same position. </li>
```

Comment:
`<li>A pinned frame always stays in the same position.</li>`

Reply:
I can't see any difference between my suggestion and yours.